lib/html-import-template-bundler.js

```diff
@@ -32,25 +24,42 @@ export function bundle(cfg) {
         let content = fs.readFileSync(file, {
           encoding: 'utf8'
         });
-  
+
         let $ = whacko.load(content);
         let name = getCanonicalName(builder, file, 'view').replace(/!view$/g, '');
   
         $('template').attr('id', name);
         let template = $.html('template');
+
         templates.push(template);
       }
     });
 
-  fs.writeFileSync(outfile, templates.join('\n'));
+  let outFilePath = writeOutput(cfg.baseURL, cfg.bundleName, templates.join('\n'), cfg.options);
 
-  if (cfg.options.inject) {
-    injectLink(outfile, baseURL, cfg.options.inject);
+  if (cfg.options && cfg.options.inject) {
+    injectLink(outFilePath, baseURL, cfg.options.inject);
   }
 
   return Promise.resolve();
 }
 
+export function writeOutput(pathName, bundleName, output, options) {
+  let outFileName = utils.getOutFileName(output, bundleName + '.html', options && options.rev);
+  let outFilePath = path.resolve(pathName, outFileName);
+
+  if (fs.existsSync(outFilePath)) {
```

Comment:
I refactored the code an put the important part, the name building, into a separate function and moved back the 8 lines.